src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -167,11 +166,11 @@ object FileMetadataService {
       propertyNameMap.get(ClientSideOriginalFilepath),
       propertyNameMap.get(ClientSideFileLastModifiedDate).map(d => Timestamp.valueOf(d).toLocalDateTime),
       propertyNameMap.get(ClientSideFileSize).map(_.toLong),
-      propertyNameMap.get(RightsCopyright.name),
-      propertyNameMap.get(LegalStatus.name),
-      propertyNameMap.get(HeldBy.name),
-      propertyNameMap.get(Language.name),
-      propertyNameMap.get(FoiExemptionCode.name),
+      propertyNameMap.get(RightsCopyright),
+      propertyNameMap.get(LegalStatus),
+      propertyNameMap.get(HeldBy),
+      propertyNameMap.get(Language),
+      propertyNameMap.get(FoiExemptionCode),
       propertyNameMap.get(ClosurePeriod).map(_.toInt),
       propertyNameMap.get(ClosureStartDate).map(d => Timestamp.valueOf(d).toLocalDateTime),
       propertyNameMap.get(FoiExemptionAsserted).map(d => Timestamp.valueOf(d).toLocalDateTime),
```

Comment:
`case class StaticMetadata(name: String, value: String)` on line 182 is no longer used with these changes so can be deleted.